libvcx/README.md

```diff
@@ -1,3 +1,16 @@
+# Deprecation notice
+This is now deprecated and receives limited maintenance support. 
+
+This project provided solution to create bindings and wrappers for other languages, 
```

Comment:
"This project used to provide C-API for creating bindings for languages other than Rust, primarily ..."